9jaswag/PostIt
# #152632081 Adhere to Redux Architecture

#### What does this PR do?
- Implements redux architecture in the app

#### Description of Task to be completed?
- adhere to redux architecture

#### How should this be manually tested?
- Check the ```client/actions``` directory to see that async actions dispatch response to store

#### Any background context you want to provide?
- None

#### What are the relevant pivotal tracker stories?
- #152632081

#### Screenshots (if appropriate)
- None

#### Questions:
- None
